spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,53 @@
+RSpec.configure do |config|
+
+  config.use_transactional_fixtures = false
+
+  config.before(:suite) do
+    if config.use_transactional_fixtures?
+
+      raise(<<-MSG)
+        Delete line `config.use_transactional_fixtures = true` from rails_helper.rb
+        (or set it to false) to prevent uncommitted transactions being used in
+        JavaScript-dependent specs.
+        During testing, the Ruby app server that the JavaScript browser driver
+        connects to uses a different database connection to the database connection
+        used by the spec.
+
+        This Ruby app server database connection would not be able to see data that
+        has been setup by the spec's database connection inside an uncommitted
+        transaction.
+        Disabling the use_transactional_fixtures setting helps avoid uncommitted
+        transactions in JavaScript-dependent specs, meaning that the Ruby app server
+        database connection can see any data set up by the specs.
+      MSG
+
+    end
+  end
+
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, type: :feature) do
```

Comment:
Do you have feature specs? You definitely don't have Capybara in this project. Be thoughtful when copying code and know exactly what you're copying and why.